action.yml

```diff
@@ -14,11 +14,8 @@ inputs:
   TWILIO_ACCOUNT_SID:
     description: 'A Twilio Account SID. Can alternatively be stored in environment'
     required: true
-  TWILIO_API_KEY:
-    description: 'A Twilio API Key. Can alternatively be stored in environment'
-    required: true
-  TWILIO_API_SECRET:
-    description: 'A Twilio API Secret. Can alternatively be stored in environment'
+  TWILIO_AUTH_TOKEN:
+    description: 'A Twilio Auth Token. Can alternatively be stored in environment'
```

Comment:
We should remove `Can alternatively be stored in environment` from all of these because it's... obvious/how it works.